app/views/templates/script/package_action.erb

```diff
@@ -115,9 +115,19 @@ handle_zypp_res_codes () {
       end
     end
   -%>
+  OUTFILE=$(mktemp)
+  trap "rm -f $OUTFILE" EXIT
+  set -o pipefail
   export DEBIAN_FRONTEND=noninteractive
   apt-get -y update
-  apt-get -o APT::Get::Upgrade-Allow-New="true" -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y <%= input("options") %> <%= action %> <%= input("package") %>
+  apt-get -o APT::Get::Upgrade-Allow-New="true" -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y <%= input("options") %> <%= action %> <%= input("package") %> 2>&1 | tee $OUTFILE
+  RETVAL=$?
+  if grep -q "Unable to locate" $OUTFILE; then
```

Comment:
This won't work if the machine's language is not english. Should we force it to be english?
```
# LC_ALL=ja_JP.UTF-8 apt-get remove foobar
パッケージリストを読み込んでいます... 完了
依存関係ツリーを作成しています
状態情報を読み取っています... 完了
E: パッケージ foobar が見つかりません
```

Reply:
Oh. Thats true. Nice catch. 
Run the "apt-get" with LANG=C?